template/package.json

```diff
@@ -5,7 +5,7 @@
   "author": "{{ author }}",
   "private": true,
   "scripts": {
-    "dev": "node build/dev-server.js",
+    "dev": "webpack-dev-server --inline --hot --progress --config build/webpack.dev.conf.js",
```

Comment:
We need to remove line 38 and 39 from `build/webpack.dev.conf.js`

```js
    // https://github.com/glenjamin/webpack-hot-middleware#installation--usage
    new webpack.HotModuleReplacementPlugin(),
```

The `--hot` is the only one needed